consensus/clique/clique.go

```diff
@@ -335,7 +335,7 @@ func (c *Clique) verifyCascadingFields(chain consensus.ChainHeaderReader, header
 	if header.GasUsed > header.GasLimit {
 		return fmt.Errorf("invalid gasUsed: have %d, gasLimit %d", header.GasUsed, header.GasLimit)
 	}
-	if !chain.Config().IsLondon(header.Number) {
+	if !chain.Config().IsBanach(header.Number) {
```

Comment:
Behavior after London and before Banach:

- Original behavior: step into `else` branch, `VerifyEip1559Header`. The original implementation of `VerifyEip1559Header` would just check gas limit then return if `baseFee == nil`.
- New behavior: step into `if` branch, `VerifyGaslimit`.

Identical.